docs/attribution/0001-analytics.rst

```diff
@@ -188,24 +200,12 @@ required.
 | ``data-download-location`` | ``primary``, ``secondary``, ``nav``, ``other``                                                              |
 +----------------------------+-------------------------------------------------------------------------------------------------------------+
 
-For all links to accounts.firefox.com use these data attributes (* indicates a required attribute):
-
-+-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Data Attribute        | Expected Value                                                                                                                                                                                                                     |
-+=======================+====================================================================================================================================================================================================================================+
-| ``data-cta-type`` *   | fxa-servicename (e.g. ``fxa-sync``, ``fxa-monitor``)                                                                                                                                                                               |
-+-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``data-cta-text``     | Name or text of the link (e.g. ``Sign Up``, ``Join Now``, ``Start Here``). We use this when the link text is not useful, as is the case with many account forms that say, ``Continue``. We replace ``Continue`` with ``Register``. |
-+-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``data-cta-position`` | Location of CTA on the page (e.g. ``primary``, ``secondary``, ``header``)                                                                                                                                                          |
-+-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
 
 **Old data-cta structure**
```

Comment:
I'm a little confused here, since does seem to be talking about an older `data-cta` pattern that exists (see doc example below your changes). 

It looks like `data-link-type` and `data-link-name` are even older maybe? Perhaps we should just list all of the outdated / obsolete attributes together?

- `data-cta-name` (when used together with `data-cta-type`, which still exists)
- `data-link-type`
- `data-link-name`

Reply:
I think I'm confused too. These data-attributes are a bit of a mess.